### ISSUE TITLE
Accessibility Review needed

### ISSUE THREAD

**Issue by owenjoseph** [hackla-engage/engage-client#129]
## Open Request
Are you familiar with [WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/) best-practices? Have you always wanted a reason to learn them? Engage needs an Accessibility Review!

The deliverable for this issue will be a report which covers:
- What is working
- What is **NOT** working
- Suggested enhancements (which we could then turn into more repo issues!)

The report can either be shared with us in this issue, or added as a new page to the [Engage Product Wiki](https://github.com/hackla-engage/Getting-Started-With-Engage/wiki).

We have specific questions regarding compatibility with screen readers and coming up with a strategy to implement related best practices. However, this report does not need to be limited to screen readers!

## New "Accessibility" Issues Tag
New Issues which arise from this review should be tagged "Accessibility," as this issue is. New issues can also reference this issue in their description.

## Background
As a constituent-facing participation platform, inclusion is a core value of the Engage design. We want no citizen to be dissuaded or impeded from having their voice heard by their government.

**Comment by TeddyCr** [comment]
Moved to a project -> https://github.com/orgs/hackla-engage/projects/5
